tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -171,11 +172,20 @@ $startTime = Get-Date
 $resourceCommands | ForEach-Object {
     $testName = $_.Name
     $script = $_.Command
+    $retry = if($null -eq $_.retry) {0} Else {$_.retry}
+    $sleep = if($null -eq $_.sleep) {30} Else {$_.sleep}
+    if($null -ne $_.Since -and "Core" -eq $PSVersionTable.PSEdition) {
+        if($PSVersionTable.PSVersion -lt [System.Version]$_.Since) {
```

Comment:
```suggestion
    if($null -ne $_.Since -and "Core" -eq $PSVersionTable.PSEdition -and $PSVersionTable.PSVersion -lt [System.Version]$_.Since) {
```

Reply:
Merged them into one line.